### DISCUSSION TITLE
API slower than the ui

### DISCUSSION THREAD

**Discussion by nephi-dev** [Comfy-Org/ComfyUI#3104] [Q&A]
Do someone know why would the API run slower than the ui itself? Here, on the UI i get a ~1.5it/s while on the api i get ~1.8s/it

**Comment by nephi-dev** [comment]
Error in my end, already figured it.

**Comment by aivrar** [comment]
Hello, can you help me with using the API? All I need to know is how you're doing it. 

I am able to use the API to create images easy enough, but I'm trying to use the API to use the image to video nodes that come with Comfy. can you help give the example if you have it handy? 

Could anyone please? I just need to run the video node and I get mapping errors that I'm not sure how to handle.

Thank you for reading.
